R/JHU.R

```diff
@@ -10,7 +14,9 @@
 #' @concept dataset
 #' @examples
 #' \dontrun{
-#' national <- JHU$new(verbose = TRUE, steps = TRUE, get = TRUE)
+#' national <- JHU$new(level = "1", verbose = TRUE, steps = TRUE, get = TRUE)
```

Comment:
I was thinking there might be some explanation in comments (like download all country level data, download all region data for the UK, download all level 2 data, see supported regions and filter for target locations that sort of thing + using memos to avoid continuous downloads).